docker-entrypoint.py

```diff
@@ -38,7 +38,7 @@ def pfioh_do(args, unknown):
 
     str_otherArgs   = ' '.join(unknown)
 
-    str_CMD = "/usr/bin/pfioh --forever %s" % (str_otherArgs)
+    str_CMD = "/usr/bin/pfioh --forever --httpResponse --storeBase=/tmp/share --createDirsAsNeeded"
```

Comment:
This I had changed long back when we needed pfioh or pman service to start according to changes we made(like --createDirasNeeded was added). But later you added the command to the dc directly and so this is not needed in Dockerfile. I must have forgotten to make the change locally and commit it on my github repo. I will remove this and do a force commit again.